msu/hooks/entity/tactical/player.nut

```diff
@@ -27,6 +27,21 @@
 		this.m.LevelUpsSpent++;
 	}
 
+	q.onHired = @(__original) function()
+	{
+		__original();
+		this.getSkills().onHired();
+
+		foreach (bro in ::World.getPlayerRoster().getAll())
+		{
+			if (bro.getID() != this.getID())
+			{
+				bro.getSkills().onOtherPlayerHired(this);
+			}
+		}
+
+	}
```

Comment:
Note that starting characters in an origin do not call this `player.onHired()` function. So, should these events be called for those characters?

Reply:
For non-random scenarios it is up to the scenario designer to manually implement interactions that his starting brothers might have.
For this implementation im purely leeching onto the vanilla onHired function and keeping within its juristiction

Reply:
Ok, I think we can do it like this. If there are concerns in the future we can resolve them at that time.